hdtv/plugins/calInterface.py

```diff
@@ -931,13 +931,15 @@ def CalsFromList(self, fname):
                 )
         return calDict
 
-    def CreateCalList(self, calDict):
+    def CreateCalList(self, calDict, sort: bool = True):
         """
         Creates a printable list of all calibrations in calDict
         <specname>: <cal0> <cal1> ...
         """
         lines = list()
-        names = sorted(calDict.keys())
+        names = calDict.keys()
```

Comment:
Is there a reason someone would want the unsorted version? I would be ok with this if there is one.

To me this seems to be a lot of code to add a configuration. Personally, I would just change the sorting from `names = sorted(calDict.keys())` to `names = sorted(names, key=hdtv.util.natural_sort_key)`

Reply:
Ok